.github/workflows/test.yml

```diff
@@ -0,0 +1,16 @@
+name: Lint and Test Charts
+on:
+  pull_request:
+    paths:
+      - "charts/**"
+jobs:
+  lint-chart:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Run chart-testing (lint)
+        uses: helm/chart-testing-action@main
+        with:
+          command: lint
+          config: .github/ct.yaml
```

Comment:
I think this file is missing. At least I don't see it being added in this pr, or was already present.

Reply:
completely forgot about the pipeline 🤦‍♂️ I'll update the PR with the full thing